### DISCUSSION TITLE
Increase limit for --max-diagnostics option past 200?

### DISCUSSION THREAD

**Discussion by digital-cuttlefish** [rome/tools#4778] [General]
Hello!

Currently, the max for the `--max-diagnostics` is 200. Is there a reason for this? 
I'm on a team that would like to use a larger number (or just have no limit and let it report all errors); could that be made possible?

**Comment by ematipico** [comment]
The main reason why there's a cap is because printing so many diagnostics degrades the performance of the tool, by a lot. 

We will provide the option to lift the limit soon
